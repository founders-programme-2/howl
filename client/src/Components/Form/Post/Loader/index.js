import React from 'react';
import { css } from '@emotion/core';
import { CircleLoader } from 'react-spinners';

const override = css``;

const Loader = ({ uploadFlag }) => {
  return (
    <div className="sweet-loading">
      <CircleLoader
        css={override}
        sizeUnit="px"
        size={15}
        color=" #c02700"
        loading={uploadFlag}
      />
    </div>
  );
};

export default Loader;
