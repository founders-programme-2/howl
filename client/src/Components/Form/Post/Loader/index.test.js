import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Loader from './index';

test('Natural', () => {
  const props = {
    uploadFlag: false,
  };
  const loader = renderer
    .create(
      <BrowserRouter>
        <Loader {...props} />
      </BrowserRouter>
    )
    .toJSON();
  expect(loader).toMatchSnapshot();
});
