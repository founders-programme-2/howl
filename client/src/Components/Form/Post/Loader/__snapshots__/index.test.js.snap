// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Natural 1`] = `
<div
  className="sweet-loading"
/>
`;
